scripts/setupvars/setupvars.sh

```diff
@@ -94,41 +94,31 @@ if [ -e $INTEL_OPENVINO_DIR/deployment_tools/tools/post_training_optimization_to
 fi
 
 if [ -z "$python_version" ]; then
-    if command -v python3.7 >/dev/null 2>&1; then
-        python_version=3.7
-        python_bitness=$(python3.7 -c 'import sys; print(64 if sys.maxsize > 2**32 else 32)')
-    elif command -v python3.6 >/dev/null 2>&1; then
-        python_version=3.6
-        python_bitness=$(python3.6 -c 'import sys; print(64 if sys.maxsize > 2**32 else 32)')
-    elif command -v python3.5 >/dev/null 2>&1; then
-        python_version=3.5
-        python_bitness=$(python3.5 -c 'import sys; print(64 if sys.maxsize > 2**32 else 32)')
-    elif command -v python3.4 >/dev/null 2>&1; then
-        python_version=3.4
-        python_bitness=$(python3.4 -c 'import sys; print(64 if sys.maxsize > 2**32 else 32)')
-    elif command -v python2.7 >/dev/null 2>&1; then
-        python_version=2.7
-    elif command -v python >/dev/null 2>&1; then
-        python_version=$(python -c 'import sys; print(".".join(map(str, sys.version_info[:2])))')
-    fi
+    python_version=$(python3 -c 'import sys; print(str(sys.version_info[0])+"."+str(sys.version_info[1]))')
 fi
 
 OS_NAME=""
 if command -v lsb_release >/dev/null 2>&1; then
     OS_NAME=$(lsb_release -i -s)
 fi
 
+python_bitness=$(python3 -c 'import sys; print(64 if sys.maxsize > 2**32 else 32)')   
 if [ "$python_bitness" != "" ] && [ "$python_bitness" != "64" ] && [ "$OS_NAME" != "Raspbian" ]; then
     echo "[setupvars.sh] 64 bitness for Python" $python_version "is requred"
 fi
 
+MINIMUM_REQUIRED_PYTHON_VERSION="3.6"    
+if [[ ! -z "$python_version" && "$(printf '%s\n' "$python_version" "$MINIMUM_REQUIRED_PYTHON_VERSION" | sort -V | head -n 1)" != "$MINIMUM_REQUIRED_PYTHON_VERSION" ]]; then
+    echo "[setupvars.sh] Unsupported Python version. Please install Python 3.6 or higher (64-bit) from https://www.python.org/downloads/"
```

Comment:
This message can be moved inside the following `if` block, it will simplify `&&` condition.

Reply:
I don't see prebuilt linux packages on python.org only OSX, e.g. https://www.python.org/downloads/release/python-379/. Perhaps this message is misleading.